viadot/flows/sap_rfc_to_adls.py

```diff
@@ -50,6 +51,8 @@ def __init__(
             name (str): The name of the flow.
             query (str): Query to be executed with pyRFC. Defaults to None.
             rfc_sep(str, optional): Which separator to use when querying SAP. If not provided, multiple options are automatically tried.
+            rfc_replacement (str, optional): In case of sep is on a columns, set up a new character to replace
+                inside the string to avoid flow breakdowns. Defaults to "-".
```

Comment:
Ah, so you mean in case the character provided in `rfc_sep` is present in a column, you replace that char with `rfc_replacement`? Normally, the way to solve this problem is by using a concept called "escaping". For example, this is how you escape special characters in Python: https://www.w3schools.com/python/gloss_python_escape_characters.asp. Have you tried doing it that way?